butterfree/load/sink.py

```diff
@@ -86,7 +94,7 @@ def validate(
     def flush(
         self, feature_set: FeatureSet, dataframe: DataFrame, spark_client: SparkClient
     ) -> List[StreamingQuery]:
-        """Trigger a write job in all the defined Writers.
+        """Trigger a build job in all the defined Writers.
```

Comment:
Is "keyword" a good word for this? I mean, even if we call the build method, I guess the docstrings can explain more than that and "write" was a better explanation of what this method does.